.github/workflows/ci.yml

```diff
@@ -134,3 +134,90 @@ jobs:
         if: always()
         working-directory: matic-cli/aws
         run: aws ec2 delete-key-pair --key-name matic-cli-ci-key
+
+  e2e-docker:
+    permissions:
+      id-token: write
+      contents: write
+    if: (github.event.action != 'closed' || github.event.pull_request.merged == true)
+    strategy:
+      matrix:
+        os: [ubuntu-20.04] # list of os: https://github.com/actions/virtual-environments
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: matic-cli
+
+      - name: Install dependencies on Linux
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt update
+          sudo apt install build-essential
+          curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash
+          sudo snap install solc
+          sudo apt install python2 jq curl
+          sudo ln -sf /usr/bin/python2 /usr/bin/python
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Install npm dependencies
+        working-directory: matic-cli
+        run: npm install --prefer-offline --no-audit --progress=false
+
+      - name: Bootstrap devnet
+        run: |
+          cd matic-cli
+          npm install --prefer-offline --no-audit --progress=false
```

Comment:
We are installing dependencies again after the `Install npm dependencies` job ?

Reply:
Good catch, thanks. Removed [here](9f2e436179eaa363000a9dc40fe931a2353fa68a)